src/uwtools/tests/drivers/test_cdeps.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+CDEPS driver tests.
+"""
+
+import datetime as dt
+import logging
+from copy import deepcopy
+from pathlib import Path
+from textwrap import dedent
+from unittest.mock import patch
+
+import f90nml  # type: ignore
+from iotaa import refs
+from pytest import fixture, mark
+
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.drivers import cdeps
+from uwtools.drivers.cdeps import CDEPS
+from uwtools.logging import log
+from uwtools.tests.support import logged
+from uwtools.tests.test_schemas import CDEPS_CONFIG
```

Comment:
The schema tests (see below) already define a CDEPS config for testing so, rather than define a new one here, it is imported from the `test_schemas` module.